hawc/apps/animal/actions/model_metadata.py

```diff
@@ -10,6 +10,10 @@ def tuple_to_dict(tuple):
     return {e[0]: e[1] for e in tuple}
 
 
+def enum_to_dict(Enum):
```

Comment:
move to common app; change argument signature `enum_to_dict(enum: Enum) -> Dict`